<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWindow</class>
 <widget class="QMainWindow" name="ServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>438</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>438</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>438</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="serverPort">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Port Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>41</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startServerBtn">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>16</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Start Server</string>
    </property>
   </widget>
   <widget class="QLabel" name="serverStatusLbl">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>405</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="onlineUsers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>261</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverIP">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>9</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>IP Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>77</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Online Clients:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>serverIP</tabstop>
  <tabstop>serverPort</tabstop>
  <tabstop>onlineUsers</tabstop>
  <tabstop>startServerBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
